julia/nfct/NFCT.jl

```diff
@@ -216,7 +216,7 @@ function Base.setproperty!(p::NFCTplan{D},v::Symbol,val) where {D}
 	elseif v == :plan
 		@warn "You can't modify the C pointer to the NFCT plan."
 	elseif v == :num_threads
-                @warn "You can't currently modify the number of threads."
+		@warn "You can't currently modify the number of threads of the NFCT plan. Use NFCT.set_num_threads(nthreads) instead."
```

Comment:
Why don't you put the code from line 343 here? I guess it might be easier to modify the calue directly

Reply:
Originally, I wanted to add the set_num_threads to the plan exactly there. However, I decided against it, since this is not a plan-specific setting but a global setting for the whole runtime.